src/bin/compute-wer.cc

```diff
@@ -128,7 +128,8 @@ int main(int argc, char *argv[]) {
       }
       num_words += ref_sent.size();
       int32 ins, del, sub;
```

Comment:
OK, I can't find my original comment.. I was saying that I did't think changing the compute-wer program is the right way to do what you are trying to do.  
But I'm not sure now. 
Where is this needed exactly?  I would have thought that 'align-text' would have given you the more detailed, word-by-word, results that are what you'd need.